### ISSUE TITLE
[Suggestion] Keyboard Navigation

### ISSUE THREAD

**Issue by ZekoTheFox** [ZekoTheFox/tf2id#1]
Self-explanatory, but to future-self I'll describe it anyways.

**Branch should be `rewrite-next`**

The website should be able to be keyboard navigated, so a loadout can be easily assembled quickly.
(Suggested by Shounic)
